### DISCUSSION TITLE
Shredos wont boot via PXE on SuperMicro machines

### DISCUSSION THREAD

**Discussion by ericrife** [PartialVolume/shredos.x86_64#295] [General]
I have setup ipxe to deliver shredos to systems on boot so I can clear partitions.  This works perfectly on a dell machine, but when i try to do this on my SuperMicro machines it just stalls and never brings up the interfaces
<img width="799" alt="Screenshot 2024-10-17 at 10 25 48 AM" src="https://github.com/user-attachments/assets/f7e49dff-9b57-44c8-9029-7f4077f9dd45">

I am using the following for PXE ( this calls the shredos pulled from the .img file)
#!ipxe
dhcp
kernel http://172.16.0.1:8080/ipxe/shredos/shredos console=tty3 loglevel=3 nwipe_options="--autonuke --method=zero --verify=off --noblank --nousb --autopoweroff"
boot

I have also tried the .iso file and called the bzImage file but it does the exact same thing.

These systems are UEFI boot, but so were the Dell machines, so not sure if that is the issue. I am thinking there is some BIOS setting that I am missing that is causing this issue, but I am at a loss.

Any help would be appreciated.

I have also tried the steps in https://github.com/PartialVolume/shredos.x86_64/discussions/148 with the same results.
<img width="795" alt="Screenshot 2024-10-17 at 11 40 59 AM" src="https://github.com/user-attachments/assets/b66d8aff-f749-4a77-84fd-b529d161e7c0">

**Comment by PartialVolume** [comment]
It could be an issue switching from frame buffer to DRM driver. Add nomodeset to the end of the kernel command line after loglevel=3

If that works, can you ALT F2 and run lspci -k so I can see what your graphics hardware is on the super micro. Thanks.

**Comment by ericrife** [comment]
This is my grub.cfg

set default="0"
set timeout="0"
set gfxpayload=1024x768x16

menuentry "shredos" {
  linux /boot/bzImage console=tty3 loglevel=3 telnetd=enable nomodeset noapic nwipe_options="--autonuke --method=zero --verify=off --noblank --nousb --autopoweroff"
}

I have tried it without telnetd as well, and still the same issue. I also tried with --nogui to nwipe and have tried it without any nwipe_options to see if comes up at all.

I have also pulled the .img file and tried this with the shredos kernel instead of bzImage and it does the same thing.

**Comment by PartialVolume** [comment]
Have you tried it with nomodeset but without noapic

**Comment by ericrife** [comment]
I did try it and it was still unsuccessful. I was able to use the BIOS
security function and destroy volumes that way, so I found a workaround.
